third_party/angular/README.md

```diff
@@ -0,0 +1,22 @@
+# Angular
+
+## URL
+
+https://github.com/angular/angular.js/releases/tag/v1.6.4
+
+## Version
+
+v1.6.4
+
+## License
+
+MIT License
+
+## License File
+
+[LICENSE](repo/angular/LICENSE.md)
+
+## Description
+Angular is a development platform for building mobile and desktop web applications using Typescript/JavaScript (JS) and other languages.
+
+This repo only includes the minified version files and LICENSE file.
```

Comment:
Change to: "Note: only the minimum set of files necessary to support the OpenThread project are included here."